src/auth/auth.ts

```diff
@@ -218,58 +227,59 @@ export const loadAuthToken = async (
     const isNewRefreshToken = !!refreshToken && refreshToken !== oldRefreshTokenMetadata.token;
 
     // for metrics, updates authToken metadata and refresh token metadata
-    authStore.update((state) => ({
+    metricStore.update((state) => ({
       ...state,
       authTokenMetadata: {
-        ...authStore.get().authTokenMetadata,
+        ...metricStore.get().authTokenMetadata,
```

Comment:
Same comment several places in this update.

```suggestion
        ...state.authTokenMetadata,
```